### ISSUE TITLE
Hovering over any of plugin cards crashes the whole website

### ISSUE THREAD

**Issue by wal-058** [elysiajs/documentation#363]
### What is the type of issue?

Something else

### What is the issue?

If I place my pointer over any of the demo plugin cards:
<img width="1433" alt="Screenshot 2024-08-06 at 12 51 08 PM" src="https://github.com/user-attachments/assets/04e3fe21-0c35-4416-952f-3bb52b00ebe1">

 the website will completely crash
<img width="950" alt="Screenshot 2024-08-06 at 12 52 40 PM" src="https://github.com/user-attachments/assets/f271f507-87db-4756-8e20-18c7812112ce">

### Where did you find it?

elysiajs.com

**Comment by fecony** [comment]
I guess you are using Opera browser that has on hover preview?

**Comment by wal-058** [comment]
No, I'm using Chrome Version 122.0.6261.129 on Mac 14.5 Sonoma (23F79) if that helps.

**Comment by fecony** [comment]
> No, I'm using Chrome Version 122.0.6261.129 on Mac 14.5 Sonoma (23F79) if that helps.

Hmm weird, Is there any additional information in that " Learn more"? 
Can you try incognito mode, turn off memory saving option or try different browser. We had reports about crashing in this case when people used Opera GX that had preview popup..

**Comment by wal-058** [comment]
I disabled all the extensions and it still didn't work. Then tried in incognito and still no sucess. What did however fix it was updating chrome to current latest of Version 127.0.6533.100.
Now, it's ok on my side but the issue still persists on that version.
